tests/test_ssm.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from cpr_sdk.ssm import get_vespa_search_adapter_from_aws_secrets
+
+
+def test_vespa_search_adapter_from_aws_secrets(
+    create_vespa_params, mock_vespa_credentials, tmpdir
```

Comment:
I did not know that `tmpdir` was a generally available [fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture)! Very happy to find that out